calcchain.go

```diff
@@ -9,16 +9,31 @@
 
 package excelize
 
-import "encoding/xml"
+import (
+	"bytes"
+	"encoding/xml"
+	"io"
+	"log"
```

Comment:
I propose to return an error to the caller of this function instead of log errors.